### ISSUE TITLE
Use unified number format independent of users locale

### ISSUE THREAD

**Issue by HenrikJannsen** [bisq-network/bisq2#2036]
There are quite different number formats for different countries (locales) [1] and the fact that the `.` and `,` is used as decimal separator is a risk for invalid input. 
Readability is also not great and as many users are used to English based international formats the confusion get bigger when confronted with the a different locale.  
I would suggest that we ignore the default local for number format but use that format:

`123 456 789.01`

- Use a '.' as decimal  separator
- Use  a space as group separator

Number of decimal places depends on the use case.

Any thoughts on that?

[1] https://docs.oracle.com/cd/E19455-01/806-0169/overview-9/index.html

**Comment by suddenwhipvapor** [comment]
I strongly agree on the dot being used as decimal separator, that is afaik the international standard for scientific papers, anyway. I understand comma is used as thousands grouping separator, but in this context it would only add to the confusion, so a space is the best candidate if a thousand grouping has to be done, maybe smaller than the space we are using right now as a word separator in this text area, if that can be done.
